soci/soci_index.go

```diff
@@ -51,9 +51,13 @@ const (
 	IndexAnnotationImageLayerDigest = "com.amazon.soci.image-layer-digest"
 	// IndexAnnotationBuildToolIdentifier is the index annotation for build tool identifier
 	IndexAnnotationBuildToolIdentifier = "com.amazon.soci.build-tool-identifier"
+	// IndexAnnotationXattrPresent is the index annotation if the layer has
+        // extended attributes
+	IndexAnnotationXattrPresent = "com.amazon.soci.xattr-present"
```

Comment:
We use this to decide if we want to disable xattrs, right? The name seems backwards.

I'm not totally sold on this. The effect of this is to disable xattrs. The state is that there are no xattrs. What do you think?
```suggestion
	IndexAnnotationXattrPresent = "com.amazon.soci.disable-xattrs"
```